Add branch-alias to composer.json

effulgentsia opened the pull request:
Per https://getcomposer.org/doc/articles/aliases.md#branch-alias, this would allow people with `"minimum-stability": "dev"` in their root composer.json to get the tip of this project's master branch if they have dependencies that require this project with a constraint of `"^1"` or similar. An example use-case for that is if someone is using Composer 2, and #22 exists on master but not in a tagged release yet.

effulgentsia committed:
Add branch-alias to composer.json

johnpicozzi commented:
Need someone to take another look at this. However, It should have a change log update with it as well.

hussainweb commented:
I think it's always a good idea to include the `branch-alias`. Now, however, this should be `2.x-dev` as 2.0.0 was just released.

effulgentsia committed:
Update branch-alias to 2.x.

effulgentsia committed:
Merge pull request #1 from oomphinc/master

Sync from upstream

effulgentsia committed:
Update CHANGELOG.md

effulgentsia commented:
I updated the alias to 2.x and added the changelog entry. I did this through the GitHub web interface, which doesn't support rebasing, so the commit history is ugly, but I hope that's ok since it probably makes sense for this to be squashed anyway if/when you merge it.

hussainweb commented:
This LGTM.
